.github/workflows/integration.yml

```diff
@@ -92,6 +92,12 @@ jobs:
           fail_ci_if_error: true
           verbose: true
 
+      - name: Upload coverage information to coveralls
+        uses: coverallsapp/github-action@v2
+        with:
+          file: ./integration/integration-${{ matrix.task }}.txt
+          flag-name: integration-${{ matrix.task }}
```

Comment:
```suggestion
          flag-name: integration-${{ matrix.task }}
          parallel: true
```

Reply:
I was considering that, although from my understanding parallel applies to jobs inside a single workflow(?)

Reply:
You need to close the parallel at the end of the workflow, but we have 1 upload job per workflow, so I'm not sure if there's any advantage of using parallel here

Reply:
I'm not sure about that, but we either need it there, or not need it in integration-trust.yml – because they are almost identical.

Also, https://github.com/coverallsapp/github-action/tree/release/v2#complete-parallel-job-example shows `parallel-finished` ?.. ¯\_(ツ)_/¯

Reply:
I will test parallel then and check how it works

Reply:
Okay, now I understand what you meant. I left over parallel in the integration-trust by accident :)

Reply:
Finally I've applied parallel in both integration and integration-trust as we use matrixes there.